WebViewControl.Avalonia/WebViewControl.Avalonia.csproj

```diff
@@ -67,8 +67,8 @@
 
   <ItemGroup>
     <!--When we upload Avalonia here, we should increase the version of both WebViewControl.Avalonia as well as ReactViewControl.Avalonia-->  
-    <PackageReference Include="Avalonia" Version="0.10.0-rc1" />  
-    <PackageReference PrivateAssets="None" Include="CefGlue.Avalonia" Version="85.4183.9" />
+    <PackageReference Include="Avalonia" Version="0.10.0-preview6" />  
```

Comment:
please revert these changes, otherwise I cant merge
already published the new cefglue packages on nuget